### ISSUE TITLE
Article vulgarizing what is TomoX

### ISSUE THREAD

**Issue by etienne-napoleone** [tomochain/docs#90]
**Estimated time:** 3h

---------------

**Why:**
Non tech-savvy investors seems to struggle grasping what's TomoX field of application and what it really do.

**What:**
Vulgarization article explaining what is TomoX. What it can do and what it can't.

**Comment by etienne-napoleone** [comment]
Assigned to: @bluebottleco 
Reviewed by: @phamvancam2104

**Comment by bluebottleco** [comment]
@phamvancam2104  Please find below my draft of the article. Let me know your thoughts / feedback. Thank you! 

I tried to write it so that anyone who has a general understanding of cryptocurrencies with no technical background can understand what TomoX is going to accomplish. 

https://medium.com/tomochain/understanding-tomox-tomochains-decentralized-exchange-protocol-2cfe540bc94b

**Comment by phamvancam2104** [comment]
Great! Thanks to @bluebottleco. Let me give it a check :)

**Comment by phamvancam2104** [comment]
Looks good to me @bluebottleco. @etienne-napoleone @anhntv2012 do you have any opinion?

**Comment by bluebottleco** [comment]
@phamvancam2104 I made the changes and thank you for looking it over! Appreciate it(:

**Comment by etienne-napoleone** [comment]
@alex9121 This article content is ready to be published, closing this issue

**Comment by anhntv2012** [comment]
Great article ^_^
Just a question: 
"This means that decentralized exchanges built on TomoChain will not need to spend time and money creating as well as auditing their smart contracts ". 
Is it clearer if "TomoChain" is replaced by "TomoX protocol"? Because I think that if they wish, they can build a different protocol on TomoChain and have a DEX on that onel.

**Comment by phamvancam2104** [comment]
@anhntv2012 I think your concern is valid. Because TomoX is uniquely integrated into TomoChain, I would rephrase the sentence into: ... decentralized exchanges built *for TomoChain based on TomoX Protocol* will not need...

**Comment by etienne-napoleone** [comment]
I updated it to ```This means that decentralized exchanges built for TomoChain using the TomoX Protocol will not need to spend time and money creating as well as auditing their smart contracts```
Is it ok with you guys?
